BranchInfo.props

```diff
@@ -3,8 +3,10 @@
     <MajorVersion>2</MajorVersion>
     <MinorVersion>1</MinorVersion>
     <PatchVersion>0</PatchVersion>
+    <StabilizePackageVersion Condition="'$(ProduceStablePackages)' != ''">$(ProduceStablePackages)</StabilizePackageVersion>
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
-    <PreReleaseLabel>preview2</PreReleaseLabel>
+    <PreReleaseLabel Condition="'$(PackagePreReleaseLabel)' != ''">$(PackagePreReleaseLabel)</PreReleaseLabel>
```

Comment:
Similar question here - should we just use `PackagePreReleaseLabel` everywhere?

Reply:
+1

Reply:
For the prereleaselabel we need to potentially change value if it is set to empty in the build definition then I want to change it to a default that is checked in, and there isn't any great way to do that when the property is set a global msbuild property.

There are a few scenarios to consider:
1) Dev scenarios where this won't be set we need to default to a checked in label.
2) Official builds that set this parameter to empty we need set it to some value, so that packages that never go stable still have a proper label. This isn't a real scenario here but it is for coreclr/corefx for things like transport packages which we will never mark stable. 
3) Official builds that set this parameter to a real label.

So I think we can cover (1) and (3) with using the same property name but we cannot cover (2) without another property name.